api/api/views/generate_teams.py

```diff
@@ -42,6 +44,5 @@ def generate_teams(self, request):
 
         serialized_team_set = TeamSetSerializer().encode(team_set)
```

Comment:
I think we forgot to call `.default` instead of `.encode`, I think this was an artifact of us changing the interface of serializers a while ago.